modules/installation-creating-gcp-lb.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-lb_{context}"]
+= Creating load balancers in GCP
+
+You must configure load balancers in Google Cloud Platform (GCP) for your
+{product-title} cluster to use. One way to create these components is
+to modify the provided Deployment Manager template.
+
+[NOTE]
+====
+If you do not use the provided Deployment Manager template to create your GCP
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and associated subnets in GCP.
+
+.Procedure
+
+. Copy the template from the *Deployment Manager template for the internal load balancer*
+section of this topic and save it as `02_lb_int.py` on your computer. This
+template describes the internal load balancing objects that your cluster
+requires.
+
+. For an external cluster, copy the template from the *Deployment Manager template for the external load balancer*
+section of this topic and save it as `02_lb_ext.py` on your computer. This
+template describes the external load balancing objects that your cluster
+requires.
+
+. Export the variables that the deployment template uses:
+
+.. Export the cluster network location:
++
+----
+$ export CLUSTER_NETWORK=`gcloud compute networks describe ${INFRA_ID}-network --format json | jq -r .selfLink`
+----
+
+.. Export the control plane subnet location:
+----
+$ export CONTROL_SUBNET=$(gcloud compute networks subnets describe <master-subnet> --region=${REGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)
```

Comment:
For the syntax both $() and are correct. In this paper some command lines are using $() and the others are using ``. It's better to keep them consistent.
Please also keep an eye on the format, it seems space or tab is missing in the front of this line.

If network variables are defined in Creating VPC network section, we can invoke it here like this:
```javascript
export CONTROL_SUBNET=`gcloud compute networks subnets describe ${HOST_PROJECT_CONTROL_SUBNET} --region=${REGION} --project ${HOST_ PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink`
```

Reply:
I'm planning to open a follow-up PR to make updates to these modules so they better align with some of our docs guidelines, and I can take a look the overall command syntax then.